meta/pyproject.toml

```diff
@@ -0,0 +1,32 @@
+[tool.poetry]
+name = "robocorp"
+version = "0.1.0"
+description = "Robocorp core libraries for Python automation"
+authors = [
+	"Ossi Rajuvaara <ossi@robocorp.com>",
+]
+readme = "README.md"
+repository = "https://github.com/robocorp/robo/"
+license = "Apache-2.0"
+packages = [{include = "robocorp", from="src"}]
+
+[tool.poetry.dependencies]
+python = "^3.9"
+robocorp-tasks = "1.0.0"
```

Comment:
I think this one should also provide the pinned `robocorp-log` version too (and when we update any version this one would be bumped up).